tests/manager/manager-executeQuery-001.phpt

```diff
@@ -73,7 +73,7 @@ object(MongoDB\Driver\Cursor)#%d (%d) {
       ["mode"]=>
       int(1)
       ["tags"]=>
-      array(0) {
+      object(stdClass)#%d (0) {
```

Comment:
I think all the else append null checks were all added at the same time (dozens of them) after I realized not everything was initialized everytime, so var_dump()ing had the tendency to crash.

I don't think I ever checked individual cases, but instead always added a fallback path.

I'd say in this particular case, tags = null would look weird. tags = array() looks best.

Why is this stdClass now? that looks weird?